portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -69,6 +72,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get the comment input from the form.
     String comment = getParameter(request, "text-input", "");
 
+    // Calculate sentiment score.
+    Document doc =
+        Document.newBuilder().setContent(comment).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
+    languageService.close();
+    System.out.println(score);
```

Comment:
Do you plan to display store the score and display it on the webpage? That would be a great next step!

Reply:
Yup, coming in the next PR :D

Reply:
Cool! :)